tests/test_admin_filters.py

```diff
@@ -208,11 +209,126 @@ def test_unpublished_filter(self):
             response_default = self.client.get(base_url)
             # filter by unpublished hide
             qs_default = response_default.context["cl"].queryset
-            response_unpublished = self.client.get(base_url + "?unpublished=1")
+            response_unpublished = self.client.get(f"{base_url}?{UNPUBLISHED_FILTER_URL_PARAM}=1")
             qs_unpublished = response_unpublished.context["cl"].queryset
             # filter by unpublished show
 
         # show all alias contents  excluding unpublished versions
         self.assertEqual(set(qs_default), set([expected_en_content]))
         # show all aliase contents including unpublished versions
         self.assertEqual(set(qs_unpublished), set([expected_unpublished]))
+
+
+class CatergoryFiltersTestCase(BaseAliasPluginTestCase):
+
+    @skipUnless(not is_versioning_enabled(), 'Test only relevant when no versioning')
+    def test_category_filter_no_verisoning(self):
+        """
+        When rendering aliascontent manager category filter, changing the category
+        should filter the results.
+        """
+        category_one = Category.objects.create(name='one')
+        alias_one = AliasModel.objects.create(
+            category=category_one,
+            position=0,
+        )
+        expected_category_one_content = AliasContent.objects.create(
+            alias=alias_one,
+            name="EN Alias Content one",
+            language="en",
+        )
+        category_two = Category.objects.create(name='two')
+        alias_two = AliasModel.objects.create(
+            category=category_two,
+            position=1,
+        )
+        expected_category_two_content = AliasContent.objects.create(
+            alias=alias_two,
+            name="EN Alias Content two",
+            language="en",
+        )
+        base_url = self.get_admin_url(AliasContent, "changelist")
+
+        with self.login_user_context(self.superuser):
+            response_default = self.client.get(base_url)
+            # category one should have a result
+            category_one_filter_response = self.client.get(f"{base_url}?alias__category__id__exact="{category_one.id}")
```

Comment:
```suggestion
            category_one_filter_response = self.client.get(f"{base_url}?alias__category__id__exact={category_one.id}")
```

This should fix the failing tests.